src/renderer/helpers/accessibility.js

```diff
@@ -0,0 +1,45 @@
+export function handleDropdownKeyboardEvent(event, target, afterElement) {
+  if (!event || !(event instanceof KeyboardEvent)) {
+    return true
+  }
+
+  let nextElement = null
+  if (event.key === 'Tab') {
+    if (afterElement) {
+      afterElement.tabindex = 0
+      afterElement.focus()
+    }
+    return false
+  } if (event.key === 'ArrowUp') {
+    nextElement = target.previousElementSibling ?? target.parentNode.lastElementChild
+
+    if (nextElement.classList.contains('listItemDivider')) {
+      nextElement = nextElement.previousElementSibling
+    }
+  } else if (event.key === 'ArrowDown') {
+    nextElement = target.nextElementSibling ?? target.parentNode.firstElementChild
+
+    if (nextElement.classList.contains('listItemDivider')) {
+      nextElement = nextElement.nextElementSibling
+    }
+  } else if (event.key === 'Home') {
+    nextElement = target.parentNode.firstElementChild
+  } else if (event.key === 'End') {
+    nextElement = target.parentNode.lastElementChild
+  }
+
+  event.preventDefault()
+
+  if (nextElement) {
+    target.setAttribute('tabindex', '-1')
+    nextElement.setAttribute('tabindex', '0')
+    nextElement.focus()
+  }
+
+  return event.key === 'Enter' || event.key === ' '
+}
+
+export function sanitizeForHtmlId(attribute) {
+  // html doesn't like spaces in ids, queryselector/css wants symbols encoded
+  return attribute.replace(/[^A-Za-z0-9]*/g, '')
```

Comment:
Does this return an empty string for languages that don't use those characters? e.g. Logographic systems?

Reply:
Good point, all ascii characters are valid for html ids (besides whitespace) so  we could include them but the issue from that would be with querySelector requiring some escaping. For example, `const allOptions = document.querySelector(`#${unspacedTitle} + ul`)` wouldn't work if it had those characters in it without some sort of escaping.

Reply:
using `CSS.escape` might work